### PR TITLE
FEAT: Added add task feature.

### PR DESCRIPTION
**Pull request by zoyatanweer**
Files to check: 
- pages/Task/Task.jsx and Task.css
- components/Modal/Modal.jsx and Modal.css
- context/TaskContext.jsx

image

### CODE REVIEW

File: src/Components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  background-color: #e9ecef;
+  border-radius: 5px;
+  border: 1px solid white;
+  /* height: 5vh; */
+  height: 8%;
+  width: 80%;
+  margin-bottom: 1.5rem;
+}
+.desc {
+  /* height: 25vh; */
```
Comment (rkapoor10) [inline src/Components/Modal/Modal.css:54]: remove height if not needed

File: src/Components/Modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  position: absolute;
+  bottom: 20px;
+}
+.add-task-btn:hover {
+  background-color: var(--yellow-secondary-color);
+  cursor: pointer;
+}
+.task-item {
+  display: flex;
+  /* justify-content: space-between; */
```
Comment (rkapoor10) [inline src/Components/Modal/Modal.css:100]: this as well

File: src/Pages/Task/Task.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  align-items: center;
+  /* justify-content: center; */
+  height: 100vh;
+
+  /* margin-bottom: 2rem; */
+}
+
+.task-section-heading {
+  /* display: block; */
+  /* display: flex; */
```
Comment (rkapoor10) [inline src/Pages/Task/Task.css:14]: same as above

File: src/Pages/Task/Task.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  /* padding: 0 1rem; */
+  /* margin: 2rem; */
+  height: 7vh;
+  width: 80vw;
+  width: 70vw;
+}
+/* .add-task:hover {
+  background-color: #e9ecef;
+  cursor: pointer;
+} */
```
Comment (rkapoor10) [inline src/Pages/Task/Task.css:58]: check for comments and remove if not required

**Review by rkapoor10 — APPROVED** [review]
Well implemented firebase. The code is well structured. Looks good to merge.

**Review by srishtigautam21 — APPROVED** [review]
The code looks good to merge
Just remove the commented code

**Comment by zoyatanweer** [comment]
thankyou guyss

**Pull request merged**